source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,128 @@
+In resolved.conf we change the entries under [Resolve] to use our own instance running at 127.0.0.1 
+::
+	sudo systemctl restart systemd-resolved
+
+We then need to stop the currently running pre-installed resolver:
+::
+	sudo systemctl disable systemd-resolved.service
+	sudo systemctl stop systemd-resolved
+
+And now we can then start using our new configuration:
+::
+	  sudo service network-manager restart
+
+And as a quick test a *dig* without specifying our Unbound server should give the same result as specifying it (with the *@127.0.0.1*)!
+::
+	dig example.com
+
+Setting up for the rest of the network
+--------------------------------------
+
+While we currently have a working instance of unbound, we need it to be reachable from within our entire network. With that comes the headache of dealing with IP addresses. It’s likely that your home router distributed local IP addresses to your devices. If this is the case (i.e. you didn’t change it by hand), the ranges should be between[`RFC1918<http://tools.ietf.org/html/rfc1918>`]:
+::
+	10.0.0.0 - 10.255.255.255 (10/8)
+	172.16.0.0 - 172.31.255.255 (172.16/12)
+	192.168.0.0 - 192.168.255.255 (192.168/16)
+
+The Unbound example config uses the 10.0.0.0/8, so that’s what we use in this example, but note that this can be a source of connectivity errors further on.
+
+Let’s look at a snippet of the example config file. The full example config is almost 1200 lines long, as the capabilities of Unbound are considerable, but we won’t need nearly as much. (If you are interested, any and all configurables can be found in the extensive manual page with *man unbound*)
+
+The example config is found at:
+::
+	/etc/unbound/unbound.conf
+
+if you open this for the first time it looks very empty. It is still usable for one machine, as this is how all the Unbound defaults are configured. It's not, however, enough for our purposes so we will add the minimal configuration options.
+
+The options that we add to the current config file to make it a "minimal usable config" are:
+::
+	server:
+            # the interface that is used to connect to the network, this means on this machine
+            interface: 0.0.0.0
+            interface: ::0
+            # addresses from the IP range that are allowed to connect to the resolver
+            access-control: 10.0.0.0/8 allow
+            access-control: 2001:DB8::/64 allow
+
+The access-control is currently configured to listen to any address on the machine, and only allow queries from the 10.0.0.0/8 IP range.
+
+To prepare our config we are going to modify the existing config in /etc/unbound/unbound.conf. 
+If you open the file we see that there is already an “include” in there. This include enables us to do DNSSEC, which allows Unbound to verify the source of the answers that it receives [LINK ?], so we want to keep this. If you don't have the files that the unclude links to, they can be created using the *unbound-anchor* command.
```

Comment:
The content of the file and the way DNSSEC is included in the config is very Debian/Ubuntu specific. Maybe show the whole supposed config file (with the include options flattened) so that readers know exactly what you are talking about and what the options do. And then you can explicitly mention the Debian/Ubuntu behavior with the include.